core/templates/dev/head/components/RatingDisplayDirectiveSpec.js

```diff
@@ -17,31 +17,29 @@
  */
 
 describe('Rating display directive', function() {
-  var elt, scope, $httpBackend, compiledElem, ctrlScope;
+  var outerScope, ctrlScope;
 
   beforeEach(module('directiveTemplates'));
   beforeEach(module('oppia', GLOBALS.TRANSLATOR_PROVIDER_FOR_TESTS));
-  beforeEach(inject(function(
-      $rootScope, $compile, _$httpBackend_, $templateCache) {
-    $httpBackend = _$httpBackend_;
-
-    var templatesHtml = $templateCache.get(
+  beforeEach(inject(function($rootScope, $compile, $templateCache) {
+    var templateHtml = $templateCache.get(
       'core/templates/dev/head/components/rating_display.html');
-    $compile(templatesHtml)($rootScope);
+    $compile(templateHtml)($rootScope);
     $rootScope.$digest();
 
-    scope = $rootScope.$new();
-    elt = angular.element(
+    outerScope = $rootScope.$new();
+
+    var elem = angular.element(
       '<rating-display rating-value="5" is-editable="true">' +
       '</rating-display>');
-    compiledElem = $compile(elt)(scope);
-    scope.$digest();
-    ctrlScope = elt[0].isolateScope();
+    var compiledElem = $compile(elem)(outerScope);
+    outerScope.$digest();
+    ctrlScope = compiledElem[0].getControllerScope();
```

Comment:
Hm, I didn't actually change any logic here, just renamed some variables. There was a $compile before (see lines 30 and 37). Both $compiles have been preserved.